web/gui/dashboard_info.js

```diff
@@ -4148,6 +4148,61 @@ netdataDashboard.context = {
         info: '<p>Network traffic received and sent by pgbouncer.</p><p><b>Received</b> - received from clients. <b>Sent</b> - sent to servers.</p>'
     },
 
+    // ------------------------------------------------------------------------
+    // CASSANDRA
+
+    'cassandra.client_requests_rate': {
+        info: 'Client requests received per second. Consider whether your workload is read-heavy or write-heavy while choosing a compaction strategy.'
+    },
+    'cassandra.client_requests_latency': {
+        info: 'Response latency of requests received per second. Latency could be impacted by disk access, network latency or replication configuration.'
+    },
+    'cassandra.cache_hit_ratio': {
+        info: 'Key cache hit ratio indicates the efficiency of the key cache. If ratio is consistently < 80% consider increasing cache size.'
+    },
+    'cassandra.cache_hit_rate': {
+        info: 'Key cache hit rate measures the cache hits and misses per second.'
```

Comment:
> Key cache hit rate measures the cache hits and misses per second

This is totally clear from the chart:
 - the title is Cache hit rate
 - dimensions hits and misses
 - units events/s

I think it is better to not add a description if it provides no additional info/value. This way we just get a false sense of finished work.

Reply:
On the other hand, dashboard info is more readable than chart titles (especially on the local dashboard), so that is ok.

Reply:
The reason I added this particular title was because we had two charts - one for ratio and one for rate, and the titles could get confusing for new users.

p.s: we are in the process of moving all info descriptions inside the info icon and they will not be visible by default. So I think we can be a bit more relaxed about how much we put in these descriptions. This change is being done in cloud currently - and once we replace local dashboard with the same UI as cloud we won't have the problem on local dashboard either.